### DISCUSSION TITLE
[Feature Request] Option to Require PIN/Password instead of Biometrics

### DISCUSSION THREAD

**Discussion by MaldroidDev** [ente-io/ente#1017] [Enhancements]
Option to Require pin or password instead of biometrics.

**Comment by Thrr-Gilag** [comment]
Have the choice that biometrics does'nt open the app seems to me usefull

**Comment by AmanRajSinghMourya** [comment]
Custom lock (PIN, password, or device lock) is now available on auth's desktop and mobile apps! You can find the latest release [here](https://github.com/ente-io/ente/releases/tag/auth-v3.1.3)
